### PR TITLE
Refactor SIMD instruction feature flags to use less bits

### PR DESCRIPTION
**Pull request by BradleyWood**

### CODE REVIEW

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
jenkins build all

**Comment by BradleyWood** [comment]
@0xdaryl Can you review?

File: compiler/x/codegen/OMRInstOpCode.hpp [hunk 1]
```diff
@@ -490,13 +503,13 @@ class InstOpCode: public OMR::InstOpCode
                {
                supported = target->supportsFeature(OMR_FEATURE_X86_AVX512F);
 
-               if (supported && flags & X86FeatureProp_EVEX128RequiresAVX512VL)
+               if (supported && flags & X86FeatureProp_AVX512VL)
```
Comment (0xdaryl) [inline compiler/x/codegen/OMRInstOpCode.hpp:506]: I know it isn't required by the C++ language rules but can you add parentheses around the `&` expressions here and elsewhere in this PR so the operator precedence is crystal clear to human readers?  I see this is the way the code was already written, but I think it is a good time to change while you're here.

**Comment by BradleyWood** [comment]
jenkins build all

@0xdaryl

**Review by 0xdaryl — APPROVED** [review]
macOS failures on X are known.  Other platform failures can't be caused by this PR.  Merging.

**Pull request merged**